man/systemd.network.xml

```diff
@@ -1629,6 +1629,15 @@ NFTSet=prefix:netdev:filter:eth_ipv4_prefix</programlisting>
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>L3mdev=</varname></term>
+        <listitem>
+          <para>A boolean. Specifies whether the rule is to directs lookups to the tables associated with VRF devices. Defaults to false.</para>
```

Comment:
Can you extend the docs. I consider myself fairly technical but this description raises more questions in my head than it answers.